app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -128,11 +166,97 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.subscription_num_columns_5:
                 setColumnNumber(5);
                 return true;
+            case R.id.subscription_filter:
+                openSubscriptionFilter();
+                return true;
             default:
                 return false;
         }
     }
 
+    private void openSubscriptionFilter() {
+        SubscriptionFilterDialog filterDialog = new SubscriptionFilterDialog(getContext(),
+                autoDownload,keepUpdated,autoDelete) {
+            @Override
+            protected void updateFilter(ArrayList<SubscriptionFilter> filterValues) {
+
+                // do some filtering work
+                // step 1: remove all existing data from current feeds
+                // Step 2: Add all where the filter value is true and assign to currentFeeds
+                currentFeeds = new ArrayList<>();
+                for(SubscriptionFilter filter: filterValues){
```

Comment:
Please add spaces:
```
for (SubscriptionFilter filter : filterValues) {
   ^                          ^               ^
```

Also, please add similar spaces to `if` statements. I know that AntennaPod is not too consistent (around 30% of the code does not use spaces) but in order to unify the style, I want all new code to have the same style.